basic_samples/Authentication/AuthorizationCodeFlow/PowerBI/OCSConnectorTest/CodedUITest1.cs

```diff
@@ -29,7 +29,7 @@ public void CodedUITestMethod1()
             this.UIMap.LaunchPowerBI();
             this.UIMap.Clickrecent();
             this.UIMap.Clickrecenturl();
-            this.UIMap.refresh();
+            // this.UIMap.refresh();
```

Comment:
Is this intended? Seems like this line should be removed or uncommented.

Reply:
readded...